core/testsuite/src/main/java/com/blazebit/persistence/testsuite/AbstractCoreTest.java

```diff
@@ -368,6 +348,10 @@ protected String singleValuedAssociationIdNullnessPath(String path, String id) {
             return path;
         }
     }
+
+    protected String resolveRegisteredFunctionName(String functionName) {
+        return getIgnoreCase(getRegisteredFunctions(), functionName);
```

Comment:
Do a map lookup in the registered functions like `STATIC_EMF_INTEGRATOR.getRegisteredFunctions(emf).get(functionName.toLowerCase())` instead.